Gems/AWSCore/Code/Tools/ResourceMappingTool/README.md

```diff
@@ -1,35 +1,35 @@
 
 # Welcome to the AWS Core Resource Mapping Tool project!
 
-## Setup aws config and credential
-Resource mapping tool is using boto3 to interact with aws services:
+## Setup aws config and credentials
+The Resource Mapping Tool uses boto3 to interact with aws services:
  * Read boto3
    [Configuration](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html) to setup default aws region.
  * Read boto3
    [Credentials](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html) to setup default profile or credential keys.
 
-Or follow **AWS CLI** configuration which can be reused by boto3 lib:
+Or follow **AWS CLI** configuration directions which can be reused by the boto3 lib:
  * Follow
    [Quick configuration with aws configure](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-config)
 
 ## Python Environment Setup Options
-### 1. Engine python environment (Including Editor)
-1. In order to use engine python environment, it requires to link Qt binaries for this tool.
+### 1. Use Engine python environment (Including Editor)
+1. In order to use the O3DE engine's python environment, this tool requires linking the Qt binaries.
 Follow cmake instructions to configure your project, for example:
    ```
    $ cmake -B <BUILD_FOLDER> -S . -G "Visual Studio 16 2019" -DLY_PROJECTS=<PROJECT_NAME>
    ```
 
-2. At this point, double check engine python environment gets setup under *<ENGINE_ROOT_PATH>/python/runtime* directory
+2. At this point, double check that the O3DE engine's python environment gets set up under *<ENGINE_ROOT_PATH>/python/runtime* directory
 
-3. Build project with **AWSCore.Editor** (or **AWSCore.ResourceMappintTool**, or **Editor**) target to generate required Qt binaries.
+3. Build the project with **AWSCore.Editor** (or **AWSCore.ResourceMappingTool**, or **Editor**) target to generate the required Qt binaries.
    ```
    $ cmake --build <BUILD_FOLDER> --target AWSCore.Editor --config <CONFIG> -j <NUM_JOBS>
```

Comment:
I know its not part of this change, but you can remove ```-J <NUM_JOBS>``` because it only works for Mac and Linux, on windows the argument is different ```/m```